### ISSUE TITLE
feat: battery percentage changed event

### ISSUE THREAD

**Issue by marcelocecin** [smashah/sulla#284]
New "change_battery" event is triggered when the battery percentage changes. Has current battery percentage and charging state as object param.

https://github.com/pedroslopez/whatsapp-web.js/commit/93a617336ac0aa15a151a5ab0fd088d497e655b7

**Comment by smashah** [comment]
lol what a random feature. but ok.

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i`);
exec(`npm run release-ci`);

var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [1.8.41](https://github.com/smashah/sulla/compare/1.8.40...1.8.41) (2020-04-06)

- Release 1.8.41 [`ceffaeb`](https://github.com/smashah/sulla/commit/ceffaeb88a1d38ca978cce69f460842809e7ec98)
- implemented `onBattery` #284 [`bcc6c99`](https://github.com/smashah/sulla/commit/bcc6c994607556d6275c059621e8c614b1c3741a)

**Comment by smashah** [comment]
@marcelocecin 

```
//listen to battery changes
  client.onBattery(battery=>console.log('BATTERY CHANGED', battery));
```

Tested to be working.
